doc/src/fix_rmc_partial.rst

```diff
@@ -0,0 +1,285 @@
+.. index:: fix rmc/partial
+
+fix rmc/partial command
+=======================
+
+Syntax
+"""""""
+
+.. code-block:: LAMMPS
+
+   fix ID group-ID rmc/partial filename.json
+
+* ID, group-ID = fix ID and group ID (as defined in :doc:`fix <fix>`)
+* filename.json = A JSON file containing the inputs for the RMC run
+
+Example
+"""""""
+
+.. code-block:: LAMMPS
+    
+    fix 1 all nvt temp 800 800 100
+    fix 2 all rmc/partial rmc_input.json
+
+An example ``rmc_input.json`` file, along with input file ``in.rmc`` and data
+file rmc_struct.data are provided in the ``examples/mc`` folder.
+
+Description
+"""""""""""
+
+This fix performs Reactive Monte Carlo Molecular Dynamics (RMCMD)
+simulations for the simultaneous treatment of the doping reaction and
+morphology evolution in doped organic semiconductors, based on
+:ref:`Verma2024<Verma2024>` and
+:ref:`Raghuraman2025<Raghuraman2025>`. The method alternates between MD
+steps (for the morphology) and MC steps where a neutral semiconductor
+and dopant molecule is chosen randomly, and the atomic charges are
+replaced to ionize the molecule. The fix is capable of capturing Integer
+Charge Transfer (ICT), where a molecule goes from neutral to fully
+ionized, and Charge Transfer Complexes (CTC), where a molecule can
+partially ionize. In addition to switching out charges, the fix can also
+switch out dihedral types, angle types and bond types of the chosen
+molecules, enabling the study of intramolecular conformational effects
+as a function of doping. The fix should used be alongside an MD fix like
+nvt or npt.
+
+.. image:: JPG/rmcmd-workflow.png
+   :align: center
+   :scale: 10%
+
+The figure above shows a schematic of the RMCMD cycle, using NVT MD as
+the fix for morphology.
+
+A step-by-step guide on running RMCMD will be made available shortly at
```

Comment:
Referring to a website is not our preferred option, since web links tend to rot and go away eventually. Also, any referral to something that *may* happen in the future is not helping users.

I thus strongly suggest to include the step-by-step guide as a "Howto" document in the "Tutorials Howto" section of the LAMMPS manual. At least a rudimentary version for this pull request that may be improved/expanded later.